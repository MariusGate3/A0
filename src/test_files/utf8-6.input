日本語は美しい言語です。