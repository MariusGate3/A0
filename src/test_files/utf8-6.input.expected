test_files/utf8-6.input: UTF-8 text
